vmware_dvs/tests/unit/utils/test_dvs_util.py

```diff
@@ -750,7 +750,8 @@ def test_creates_network_map_from_conf(self, *args):
             vmware_conf.vsphere_login,
             vmware_conf.vsphere_password,
             vmware_conf.api_retry_count,
-            vmware_conf.task_poll_interval)
+            vmware_conf.task_poll_interval,
+            pool_size=100)
```

Comment:
Let's move pool_size constant it in config file(/vmware_dvs/common/config.py,  vmware_opts) and use smth like:
pool_size = vmware_conf.connections_pool_size